traffic_ops/traffic_ops_golang/config/config.go

```diff
@@ -292,8 +292,16 @@ func ParseConfig(cfg Config) (Config, error) {
 		if cfg.URL, err = url.Parse(listen); err != nil {
 			invalidTOURLStr = fmt.Sprintf("invalid Traffic Ops URL '%s': %v", listen, err)
 		}
+
 		cfg.KeyPath = cfg.GetKeyPath()
+		if cfg.KeyPath == "" {
+			return Config{}, fmt.Errorf("key in %s cannot be blank", listen)
+		}
+
 		cfg.CertPath = cfg.GetCertPath()
```

Comment:
seems you picked up a change from my other PR #3480 that was reverted.   This check is done in the main package instead so the error doesn't get lost.